xcdat/regridder/regrid2.py

```diff
@@ -564,17 +608,12 @@ def _get_dimension(input_data_var, cf_axis_name):
 
 
 def _get_bounds_ensure_dtype(ds, axis):
-    cf_keys = CF_ATTR_MAP[axis].values()
-
     bounds = None
 
-    for key in cf_keys:
-        try:
-            name = ds.cf.bounds[key][0]
-        except (KeyError, IndexError):
-            pass
-        else:
-            bounds = ds[name]
+    try:
+        bounds = ds.bounds.get_bounds(axis)
+    except KeyError:
+        pass
 
```

Comment:
I simplified this logic by using `ds.bounds.get_bounds()` instead of `ds.cf.bounds`.